desktop/packages/mullvad-vpn/scripts/verify-translations-format.ts

```diff
@@ -69,6 +71,10 @@ function checkFormatSpecifiers(translation: GetTextTranslation): boolean {
     .every((result) => result);
 }
 
+function isAllowedTag(tag: string): tag is ValueOfArray<typeof ALLOWED_TAGS> {
+  return ALLOWED_TAGS.some((allowedTag) => tag === allowedTag);
```

Comment:
Nitpicking, but slightly simpler to just use `includes()`:

```suggestion
  return ALLOWED_TAGS.includes(tag);
```

Since `some()` is just there to consume a custom logic for the check.